### PR TITLE
MP-3612 Added POST multi-colli shipment endpoint.

### PR DESCRIPTION
**Pull request by NickVries**
# Related Issues
- https://myparcelcombv.atlassian.net/browse/MP-3612

### CODE REVIEW

File: specification/paths/MultiColliShipment.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "post": {
+    "tags": [
+      "Multi-colli shipment"
```
Comment (M4tini) [inline specification/paths/MultiColliShipment.json:4]: I would say keep it plural like all other endpoints: `Multi-colli shipments`

File: specification/paths/MultiColliShipment.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            "data"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "data": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Shipment"
+              }
+            }
```
Comment (M4tini) [inline specification/paths/MultiColliShipment.json:26]: What about the `meta`?
I think we should support the same meta as the shipment endpoint (which will get applied to all shipments).

**Review by M4tini — CHANGES_REQUESTED** [review]
Please use `/specification/tags.json` to order them below "Shipments"

Also, I'm missing the "parcel number" mentioned in the task. Thinking about it maybe the microservice should be responsible for assigning this number, but it should at least be communicated in the response.

File: specification/paths/MultiColliShipment.json (hunk 1 above)
Comment (NickVries) [reply specification/paths/MultiColliShipment.json:4]: Makes sense! :ok_hand:

File: specification/paths/MultiColliShipment.json (hunk 2 above)
Comment (NickVries) [reply specification/paths/MultiColliShipment.json:26]: I initially had it in there, but didn't know if it was a requirement. I can add it back.

**Comment by NickVries** [comment]
> Please use `/specification/tags.json` to order them below "Shipments"
> 
> Also, I'm missing the "parcel number" mentioned in the task. Thinking about it maybe the microservice should be responsible for assigning this number, but it should at least be communicated in the response.

Didn't know that `tags` thing. Will do. I was also thinking about parcel number, I can include it in the response.

**Review by M4tini — APPROVED** [review]

**Pull request merged**